### PR TITLE
Added  New upgrade test from v3.1.0 to latest

### PR DESCRIPTION
**Pull request by anpanigr**
Usecase Description 
 
Operator upgrade from 3.1.0 to latest.
 Install 3.1.0 Operator from GitHub chart repository and create a domain.
 Deploy an application to the cluster in domain and verify the application can be 
      accessed while the operator is upgraded and after the upgrade.
  Upgrade operator with latest Operator image and verify CRD version and image are updated
     and the domain can be managed by scaling the cluster using operator REST api.
 
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/3129

### CODE REVIEW

**Review by vanajamukkara — APPROVED** [review]
LG

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java [hunk 1]
```diff
@@ -114,10 +112,9 @@ public static void init() {
    */
   @Test
   @DisplayName("Upgrade Operator from 2.5.0 to latest")
-  @MustNotRunInParallel
   public void testOperatorUpgradeFrom2_5_0(@Namespaces(3) List<String> namespaces) {
     this.namespaces = namespaces;
-    upgradeOperator("2.5.0", false);
+    upgradeOperator("2.5.0", TestConstants.OLD_DEFAULT_EXTERNAL_SERVICE_NAME_SUFFIX, false);
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java:117]: use static imports

**Review by sankarpn — CHANGES_REQUESTED** [review]
a minor change

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorUpgrade.java]: done

**Review by sankarpn — APPROVED** [review]
Great.

**Pull request merged**